tests/MongoDB.Driver.Tests/Specifications/Runner/MongoClientJsonDrivenTestRunnerBase.cs

```diff
@@ -62,7 +62,7 @@ public abstract class MongoClientJsonDrivenTestRunnerBase
         private string DatabaseName { get; set; }
         private string CollectionName { get; set; }
 
-        private IDictionary<string, object> _objectMap = null;
+        private Dictionary<string, object> _objectMap;
```

Comment:
The changes made to this file related to the object map are *not* needed.

There are other changes that *are* needed...